### PR TITLE
nodejs-jquery.cookie 1.4.1

### PR DESCRIPTION
**Pull request by dLobatog**
After https://github.com/theforeman/foreman/pull/3433#issuecomment-210983567 merge

### CODE REVIEW

File: nodejs-jquery.cookie/nodejs-jquery.cookie.spec [hunk 1]
```diff
@@ -0,0 +1,62 @@
+%global npm_name jquery.cookie
+
+Name: nodejs-%{npm_name}
+Version: 1.4.1
+Release: 1%{?dist}
+Summary: A simple, lightweight jQuery plugin for reading, writing and deleting cookies.
+License: MIT
+Group: Development/Libraries
+URL: git://github.com/carhartl/jquery-cookie.git
```
Comment (domcleal) [inline nodejs-jquery.cookie/nodejs-jquery.cookie.spec]: http?

Comment (domcleal) [inline nodejs-jquery.cookie/nodejs-jquery.cookie.spec]: https://github.com/js-cookie/js-cookie ?

File: nodejs-jquery.cookie/nodejs-jquery.cookie.spec [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+%exclude %{nodejs_sitelib}/%{npm_name}/Gruntfile.js
+%exclude %{nodejs_sitelib}/%{npm_name}/*.json
+%exclude %{nodejs_sitelib}/%{npm_name}/*.md
+
+
+%files doc
+%doc %{nodejs_sitelib}/%{npm_name}/README.md
+%doc %{nodejs_sitelib}/%{npm_name}/CONTRIBUTING.md
+%doc %{nodejs_sitelib}/%{npm_name}/CHANGELOG.md
+%doc %{nodejs_sitelib}/%{npm_name}/MIT-LICENSE.txt
```
Comment (domcleal) [inline nodejs-jquery.cookie/nodejs-jquery.cookie.spec]: move to the main %files list

**Comment by dLobatog** [comment]
Rebased

File: nodejs-jquery.cookie/nodejs-jquery.cookie.spec [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+Summary: A simple, lightweight jQuery plugin for reading, writing and deleting cookies.
+License: MIT
+Group: Development/Libraries
+URL: http://github.com/carhartl/jquery-cookie
+Source0: http://registry.npmjs.org/jquery.cookie/-/jquery.cookie-1.4.1.tgz
+
+BuildRequires: nodejs-devel
+BuildRequires: nodejs-packaging
+Requires: npm(jquery) >= 1.2.0
+bUILdArch: noarch
```
Comment (domcleal) [inline nodejs-jquery.cookie/nodejs-jquery.cookie.spec]: BuildArch

File: comps/comps-foreman-fedora24.xml [hunk 4]
```diff
@@ -33,6 +33,8 @@
       <packagereq type="default">foreman-installer</packagereq>
       <packagereq type="default">rubygem-puppet-strings</packagereq>
 
+      <packagereq type="default">nodejs-jquery.cookie</packagereq>
```
Comment (domcleal) [inline comps/comps-foreman-fedora24.xml]: Missing -doc subpackage, ditto in all PRs I think.

**Comment by dLobatog** [comment]
Repushed with -doc package and typo fixed

**Comment by domcleal** [comment]
Merged as 5df3851, thanks @dLobatog.

**Pull request closed without merging**